### ISSUE TITLE
Update the fixtures of valid Princeton records with the removal of the enrichment fields 876 i and 876 h

### ISSUE THREAD

**Issue by christinach** [pulibrary/marc_liberation#1043]
It indexes use_statement with an odd value: `"use_statement"=>"Oversize M23.L5S6 1973q"`
{"location_code"=>"annex$stacks", "location"=>"Forrestal Annex - Annex Stacks", "library"=>"Forrestal Annex", "call_number"=>"stacks M23.L5S6 1973q Oversize", "call_number_browse"=>"M23.L5S6 1973q Oversize", "items"=>[{"holding_id"=>"22242008800006421", "id"=>"23242008790006421", "status_at_load"=>"1", "use_statement"=>"Oversize M23.L5S6 1973q", "barcode"=>"32101030094724", "copy_number"=>"10"}]}

part of #823

**Comment by christinach** [comment]
The enrichment fields in alma specifies 876 | h as: call_number subfield. This is why based on the current codebase it indexes the call number. This should be fixed.

In voyager an 876 | h would expect to have 'In library use', 'Supervised Use' or nothing. The logic is based on the locations in subfield z. Do we need to add 876 enrichment field | z? 

Update: We discussed this in the alma checkin and we removed from all the publishing jobs the `876 | h` and `876 | i` subfields.
